source/OptimaJet.DWKit.StarterApplication/Migrations/AppDbContextModelSnapshot.cs

```diff
@@ -257,6 +257,32 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                     b.ToTable("Products");
                 });
 
+            modelBuilder.Entity("OptimaJet.DWKit.StarterApplication.Models.ProductArtifact", b =>
```

Comment:
Did you rebase your migrations on master before submitting the PR?  Yesterday, I had to make some changes to the StoreId to be optional.  Sorry. :-(